### PR TITLE
Add Credo

### PR DESCRIPTION
**Pull request by jomei**

### CODE REVIEW

**Comment by jomei** [comment]
`/home/travis/.travis/job_stages: line 53: $'\342\200\224': command not found`

yobaniy v rot

**Comment by begor** [comment]
@jomei 
chisti blyat

File: .travis.yml [hunk 1]
```diff
@@ -5,9 +5,22 @@ services:
...
+
 otp_release:
   - 19.0
 
+env:
+ global:
+   - MIX_ENV=test
+
+before_script:
+  - rvm install ruby-2.3.1
```
Comment (begor) [inline .travis.yml]: WAT

Comment (jomei) [reply .travis.yml]: this is required for pronto

**Comment by jomei** [comment]
Leave github integration for a while..Maybe later. Let's merge credo only

**Pull request merged**